### DISCUSSION TITLE
Please consider removing the auto-query feature

### DISCUSSION THREAD

**Discussion by feralto** [Innofactor/XrmToolBox.BulkDataUpdater#135] [Ideas]
I love this tool, I use it all the time to get data you can't get from Advanced Find but please consider removing the feature that automatically runs the last query when the tool is opened. This feature is frustrating if you've run a query that takes a long time to complete and, if you input new Fetch XML, it appears to be running two queries at once, slowing both.

**Comment by rappen** [comment]
Totally agree on that.
It will remember the query, but not execute 👍

**Comment by feralto** [comment]
Thank you!

I'll keep this email forever!

Jeff
-------- Original Message --------
